ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>AD_UNIT_ID_FOR_BANNER_TEST</key>
+	<string>ca-app-pub-3940256099942544/2934735716</string>
+	<key>AD_UNIT_ID_FOR_INTERSTITIAL_TEST</key>
+	<string>ca-app-pub-3940256099942544/4411468910</string>
+	<key>CLIENT_ID</key>
+	<string>327219256828-cc6pbt663ltp1ef2g7t5bmret1tae8ko.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.327219256828-cc6pbt663ltp1ef2g7t5bmret1tae8ko</string>
+	<key>API_KEY</key>
+	<string>AIzaSyDeWY0v5V_IFq6pwSJn5diN-5m8RpqpuHE</string>
+	<key>GCM_SENDER_ID</key>
+	<string>327219256828</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>com.svbootcamp.bridgeme</string>
+	<key>PROJECT_ID</key>
+	<string>bridgeme-83475</string>
+	<key>STORAGE_BUCKET</key>
+	<string>bridgeme-83475.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<true/>
+	<key>IS_ANALYTICS_ENABLED</key>
+	<false/>
+	<key>IS_APPINVITE_ENABLED</key>
+	<false/>
+	<key>IS_GCM_ENABLED</key>
+	<true/>
+	<key>IS_SIGNIN_ENABLED</key>
+	<true/>
+	<key>GOOGLE_APP_ID</key>
+	<string>1:327219256828:ios:f2aeba822ae3a516</string>
+	<key>DATABASE_URL</key>
+	<string>https://bridgeme-83475.firebaseio.com</string>
+</dict>
+</plist>
```

Comment:
I think that it is okay to keep this, because this file is created automatically.
To be honest, grade files and json files of google play services are same case.